docker/serve/Dockerfile

```diff
@@ -21,13 +21,13 @@ ENV PATH="/opt/conda/bin:$PATH"
 RUN export FORCE_CUDA=1
 
 # TORCHSEVER
-RUN pip install torchserve torch-model-archiver
+RUN ["/bin/bash", "-c", "pip install --no-cache-dir torchserve torch-model-archiver"]
```

Comment:
Is the modification necessary? We may leave comments here to indicate the reason

Reply:
This will result in smaller image size. Pip caches do not need to be kept in Docker images. [https://stackoverflow.com/questions/45594707/what-is-pips-no-cache-dir-good-for](this problem on StackOverflow)

Reply:
Great! can you leave this comments above the command to remind other developers?

Reply:
Sure, it's done.